lib/trino-parquet/pom.xml

```diff
@@ -222,6 +222,29 @@
     </dependencies>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <compilerArgs combine.children="append">
+                            <compilerArg>${extraJavaVectorArgs}</compilerArg>
```

Comment:
I got rid of `maven-compiler-plugin` from here but ended up having to keep the `maven-javadoc-plugin` config here because adding that to the parent pom breaks modules which are not on java 22 yet.

Reply:
Maybe we should just special-case the configuration for the modules that are not on Java 22 (I believe it's just the grammar, client API and CLI).

Reply:
Right now it feels more straightforward to keep the extra configuration in parquet, maybe we can change this when we start having more modules with this override and then the ones which are not on Java 22.